lib/config/config.js

```diff
@@ -135,30 +123,18 @@ function createConfig(opts, logger) {
 
 class Config {
   constructor(opts, logger) {
-    this.disableMetricsRegExp = [];
-    this.ignoreUrlStr = [];
-    this.ignoreUrlRegExp = [];
-    this.ignoreUserAgentStr = [];
-    this.ignoreUserAgentRegExp = [];
-    this.elasticsearchCaptureBodyUrlsRegExp = [];
-    this.transactionIgnoreUrlRegExp = [];
-    this.sanitizeFieldNamesRegExp = [];
-    this.ignoreMessageQueuesRegExp = [];
+    // TODO: remove this call once config and logging-preamble tests are refactored
+    readEnvOptions();
```

Comment:
Note for reviewer: env should be read once but some tests are setting env dinamically to tests different scenarios so its mandatory to read again. In a follow up PR I will move the tests to fixtures (using `runTestFixtures`) so we have better control of the env.

Reply:
FWIW, I'm okay with env options being read from `process.env` when `Config`s constructor is called. I think it should be okay for a *user* to tweak `process.env.*` before importing and starting the APM agent.